### PR TITLE
Release changes for 2600

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
Release changes for 2600

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by SHANDCRUZ — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 ## Build Status 
 
-OV Version | 5.60 | 5.50 | 5.40 | 5.30 |
+OV Version | 6.0 | 5.60 | 5.50 | 5.40 |
```
Comment (VenkateshRavula) [inline README.md:5]: mark the OV version as 6.00

File: README.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
 | ------------- |:-------------:| -------------:| -------------:| -------------:|
-SDK Version/Tag | [Master](https://github.com/HewlettPackard/oneview-sdk-ruby/tree/master) | [v5.17.0](https://github.com/HewlettPackard/oneview-sdk-ruby/releases/tag/v5.17.0) | [v5.16.0](https://github.com/HewlettPackard/oneview-sdk-ruby/releases/tag/v5.16.0) | [v5.15.0](https://github.com/HewlettPackard/oneview-sdk-ruby/releases/tag/v5.15.0) |
+SDK Version/Tag | [Master](https://github.com/HewlettPackard/oneview-sdk-ruby/tree/master) | [v5.18.0](https://github.com/HewlettPackard/oneview-sdk-ruby/releases/tag/v5.18.0) | [v5.17.0](https://github.com/HewlettPackard/oneview-sdk-ruby/releases/tag/v5.17.0) | [v5.16.0](https://github.com/HewlettPackard/oneview-sdk-ruby/releases/tag/v5.16.0) |
```
Comment (VenkateshRavula) [inline README.md:7]: change master to 6.0.0 and change the respective url. I kept it as master by mistake in previous release, please correct now.

File: README.md [hunk 3]
```diff
@@ -72,7 +72,7 @@ client = OneviewSDK::Client.new(
   logger: Logger.new(STDOUT),         # This is the default
   log_level: :info,                   # This is the default
   domain: 'LOCAL',                    # This is the default
-  api_version: 2400                   # Defaults to appliance's max API version which is API version of OneView 5.60
+  api_version: 2600                   # Defaults to appliance's max API version which is API version of OneView 5.60
```
Comment (VenkateshRavula) [inline README.md:75]: change OneView version to 6.00

Comment (yuvirani) [reply README.md]: Done

**Review by VenkateshRavula — APPROVED** [review]

**Pull request merged**